tbapy/main.py

```diff
@@ -463,6 +461,13 @@ def delete_event_matches(self, data):
         """
         return self._post('event/%s/matches/delete', json.dumps(data))
 
+    def delete_all_event_matches(self):
```

Comment:
I found this new write api endpoint while reading through TBA's code. This is not listed in their API docs.

Reply:
@NikhilNarayana Oh hey, neat. Might want to open an issue on TBA about that if it hasn't already been opened.

I might be more into calling this `delete_event_matches()`... not a big deal though.

Reply:
That function name does look nicer, I might change it before I commit and update to the README.